sar_core/UserAttention[] attention
